draft-ietf-scitt-architecture.md

```diff
@@ -326,6 +326,16 @@ The SCITT architecture consists of a very loose federation of Transparency Servi
 In order to accommodate as many Transparency Service implementations as possible, this document only specifies the format of Signed Statements (which must be used by all Issuers) and a very thin wrapper format for Receipts, which specifies the Transparency Service identity and the agility parameters for the Signed Inclusion Proofs.
 Most of the details of the Receipt's contents are specified in the COSE Signed Merkle Tree Proof document {{-COMETRE}}.
 
+{{fig-concept-relationship}} illustrates entities and processes that comprise a Transparency Service independent of any one use case.
+There are three main entities and their associated processes in SCITT:
+
+* Issuers that use Statements about Artifacts and their credentials to create Signed Statements
+* Transparency Services that register Signed Statements and Transparent Statements and in doing produce Receipts
```

Comment:
```suggestion
* Transparency Services that evaluate Signed Statements against Registration Policies, producing Receipts upon successful Registration
```

Reply:
I disagree with this loss of capabilities.  A ledger should be able to take a transparent statement from another ledger and handle it as well.

Reply:
While accurate, a Signed Statement can be a previously Signed Statement with an attached receipt (Transparent Statement), it's not what [the diagram currently shows](https://ietf-wg-scitt.github.io/draft-ietf-scitt-architecture/draft-ietf-scitt-architecture.html#section-4).

The diagram shows a single signed statement being placed on one, two, or more Transparency Services. Each TS produces a receipt, which can be stapled into a single Signed Statement with two receipts. 

The [current text](https://ietf-wg-scitt.github.io/draft-ietf-scitt-architecture/draft-ietf-scitt-architecture.html#section-4), nor the diagram cover the scenario of a Transparent Statement being re-registered. That could be a separate PR if we want to load the diagram with more lines.

The other suggested change describes the Signed Statement must be first evaluated against a registration policy before it's successfully registered.

Reply:
Can we get a vote for how to resolve this? 
If the goal is to make the text describe the diagram, the above suggestion hits that mark. 
If we're adding other, valid capabilities, but not reflected in the diagram, we're creating another hole we'll need to patch.

Reply:
I am not sure what I missed with the original request to add figure text and the diagram description, but I intended to resolve that. So I am willing to take in the suggestion.

That said, I have switched GitHub accounts as I terminated my account NIST, so I may need to reopen this PR and touch up accordingly. Should I proceed wit that?